psydac/api/__init__.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: UTF-8 -*-
 
 from psydac.api import ast
+from psydac.api import old_ast
```

Comment:
Do we need to keep the old AST definitions?

Reply:
no not really